### PR TITLE
add: backend paddle corrcoef

### LINKED ISSUE #13817: corrcoef

### PR DESCRIPTION
**Pull request by trantuantdt**
Close #13817

### CODE REVIEW

**Comment by MahmoudAshraf97** [comment]
Hi, can you explain what are the changes in the testing function and why are they needed?

**Comment by trantuantdt** [comment]
Hi @MahmoudAshraf97. if the input array has a column where all rows have the same value, that column's variance is 0. All the values divided by 0 will be replaced by nan. There will be incorrect results returned by Jax and Paddle backend. So I replace all the columns and rows ( in case "rowvar" is False) which have the same value in every position.

**Comment by trantuantdt** [comment]
I can't fork from the PaddlePaddle branch. It doesn't exist anymore.

File: ivy_tests/test_ivy/test_functional/test_experimental/test_core/test_statistical.py [hunk 1]
```diff
@@ -173,22 +173,47 @@ def test_quantile(
...
+    )
+
+    dtype, value = dtype_and_x
+    x = value[0]
+    y = value[1]
+    dim_1 = x.shape[0]
+    dim_2 = x.shape[1]
+    for i in range(dim_1):
+        if len(np.unique(x[i, :])) == 1:
+            x[i] = np.random.uniform(-100, 100, dim_2)
```
Comment (CatB1t) [inline ivy_tests/test_ivy/test_functional/test_experimental/test_core/test_statistical.py:201]: The usage of `np.random` is prohibited, it doesn't provide a shrinking feature, which makes fixing a failing test harder. We should be using the strategies provided by both Ivy and Hypothesis to generate our data.

**Review by CatB1t — CHANGES_REQUESTED** [review]

File: ivy_tests/test_ivy/test_functional/test_experimental/test_core/test_statistical.py (hunk 1 above)
Comment (trantuantdt) [reply ivy_tests/test_ivy/test_functional/test_experimental/test_core/test_statistical.py:201]: Is that ok to replace all rows and columns which have the same value at every position here?

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

**Comment by trantuantdt** [comment]
I would like to continue working on this PR

**Review by CatB1t — CHANGES_REQUESTED** [review]
Hello @trantuantdt , the tests seems to be passing fine for all backends on master without changing the behaviour or the generated data, I don't think we should be changing the strategy, please try to sync your branch with the Ivy's master branch. We should try and fix the `paddle` backend to return the same resutls from the ground truth framework `tensorflow`

**Pull request closed without merging**